src/rviz/TSRMarker.cpp

```diff
@@ -0,0 +1,14 @@
+#include <aikido/rviz/TSRMarker.hpp>
+
+using dart::dynamics::SimpleFrame;
+
+namespace aikido {
+namespace rviz {
+
+TSRMarker::TSRMarker(std::vector<std::unique_ptr<SimpleFrame>> tsrFrames)
```

Comment:
We could use rvalue reference (`std::vector<std::unique_ptr<SimpleFrame>>&& tsrFrames`) here to prevent unnecessary copy.